app/models/request.rb

```diff
@@ -7,6 +7,7 @@ class Request < ApplicationRecord
   validates :content, :user, :duedate, presence: true
 
   enum status: { pending: 0, accepted: 1, rejected: 2, commented: 3 }
+  enum open_closed: { open: 0, closed: 1 }
```

Comment:
I thought about that, but if I did so, user cannot use open/closed and pending/accepted... at the same time as a search query.
I think it is not convenient, how do you think?